ecs/world.go

```diff
@@ -61,7 +45,7 @@ func (w *World) RemoveEntity(entity *Entity) {
 }
 
 // AddSystem adds a new System to the World, and then sorts them based on Priority
-func (w *World) AddSystem(system Systemer) {
```

Comment:
I like having both; for a lot of things, the (currently) `BasicSystem` is also kind of nice. :smile: